test/e2e/flowaggregator_test.go

```diff
@@ -259,13 +266,35 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 						t.Errorf("Record does not have correct egressNetworkPolicyNamespace")
 					}
 				}
-				// Check the bandwidth using octetDeltaCount in data record.
 				if checkBandwidth {
-					checkBandwidthFromRecord(t, record, bandwidth)
+					TotalBytesFromSource = math.Max(getTotalBytesFromRecord(t, record), TotalBytesFromSource)
 				}
 			}
 		}
 	}
+	// Check average bandwidth using octetTotalCount in data record.
+	if checkBandwidth {
+		// bandwidth from iperf output
+		bwSlice := strings.Split(bandwidth, " ")
+		iperfBandwidth, err := strconv.ParseFloat(bwSlice[0], 64)
+		if err != nil {
+			t.Errorf("Error in converting iperf bandwidth to float64 type")
+		}
```

Comment:
`require.NoError(err, "Error when converting iperf bandwidth to float64 type")`

Reply:
Thanks, addressed.